atat_provisioning_wizard_api.yaml

```diff
@@ -455,19 +477,26 @@ components:
             error_map:
               type: object
               description: 'Maps form input IDs to validation error messages so that clients can display in-line errors'
+    FileMetadataSummary:
+      type: object
+      description: Metadata describing an uploaded file
+      properties:
+        id:
```

Comment:
Does this have any prescribed format?

Reply:
I'll add that it's a UUID. In practice, it will be the S3 Object Key name, which can be anything unique within a Bucket, although we'll want to use the same UUIDs across all three Buckets.